src/OSPSuite.Presentation/Presenters/SimulationPredictedVsObservedChartPresenter.cs

```diff
@@ -87,7 +87,8 @@ protected override void AddRunResultToChart()
          addPredictedVsObservedToChart(_simulation.ResultsDataRepository, (column, curve) =>
          {
             curve.Description = curve.Name;
-            curve.Name = column.PathAsString;
+            SelectColorForPath(curve.Name);
```

Comment:
This code adds the distinct colors for curves with different names. In other places the cache uses the DataColumn path, but I don't think it's possible here. The column path is going to be the same for multiple observations, but we want to see distinct colors for distinct observations.